### PR TITLE
Response codes and descriptions

### PR DESCRIPTION
**Pull request by KimSchneider**
I've made some changes regarding the description of response codes, requests and responses.
According to spec desciptions support markdown, but the template didn't.

### CODE REVIEW

**Comment by wdullaer** [comment]
Thanks. I'll try to review this still somewhere this week.

File: templates/resource.nunjucks [hunk 1]
```diff
@@ -59,12 +59,16 @@
...
-  {%- for response in method.responses -%}
-  {{ comma() }} {{ response.code }}
-  {%- endfor -%}
-</p>
+<dl>
+{% for response in method.responses %}
+  <dt>{{ response.code }}</dt>
+  <dd>
+     {% markdown -%}
+        {{ response.description}}
```
Comment (wdullaer) [inline templates/resource.nunjucks]: I know this is super nitpicky, but there should be a space after description

File: templates/resource.nunjucks [hunk 2]
```diff
@@ -59,12 +59,16 @@
...
 <p><strong><em><big>Possible Responses</big></em></strong></p>
-<p>
-  {%- set comma = joiner() -%}
-  {%- for response in method.responses -%}
-  {{ comma() }} {{ response.code }}
-  {%- endfor -%}
-</p>
+<dl>
+{% for response in method.responses %}
+  <dt>{{ response.code }}</dt>
```
Comment (wdullaer) [inline templates/resource.nunjucks]: To keep the layout consistent, we should maybe use the same for as for the other keys (as defined in `parameter-table.nunjucks`

```nunjucks
<p>
  <strong>{{ response.code }}</strong>
  {% markdown -%}
    {{ response.description }}
  {%- endmarkdown %}
</p>
```

You can massively screw up the layout with markdown here, but that's the user's own problem I suppose

File: templates/resource.nunjucks (hunk 1 above)
Comment (KimSchneider) [reply templates/resource.nunjucks]: You are right, would have said the same ^^

File: templates/resource.nunjucks (hunk 2 above)
Comment (KimSchneider) [reply templates/resource.nunjucks]: Alright, will fix that tomorrow. 

You are right, one could screw up the layout. But as you said, that's the user's own problem. We use it mainly to highlight params in `inline code` style and display line breaks.

**Pull request merged**